plugin/testcaplugin/main.go

```diff
@@ -0,0 +1,61 @@
+//go:build full || e2e
+
+package main
```

Comment:
any plan to support it in some other plugin?

Reply:
I did it this way because I was just trying to quickly get an e2e test working rather than building a fully working plugin. This should also provide working example code for anyone that would need to get started with this new feature. The plugin that is being working on by us is pretty specific and closed source. If there is any other parts that can be made open source, I will be sure to make another pull request.